src/docs/development/ui/layout/index.md

```diff
@@ -1,1191 +1,52 @@
-  * `GridView.extent` allows you to specify the maximum pixel width of a tile
-{% comment %}
-* Use `MediaQuery.of(context).orientation` to create a grid that changes
-  its layout depending on whether the device is in landscape or portrait mode.
-{% endcomment %}
-
-{{site.alert.note}}
-  When displaying a two-dimensional list where it's important which
-  row and column a cell occupies (for example,
-  it's the entry in the "calorie" column for the "avocado" row), use
-  [Table]({{api}}/widgets/Table-class.html) or
-  [DataTable]({{api}}/material/DataTable-class.html).
-{{site.alert.end}}
-
-#### Examples (GridView)
-{:.no_toc}
-
-<div class="row">
-<div class="col-lg-6" markdown="1">
-  {% asset ui/layout/gridview-extent.png class="mw-100" alt="A 3-column grid of photos" %}
-  {:.text-center}
-
-  Uses `GridView.extent` to create a grid with tiles a maximum 150 pixels wide.
-
-  **App source:** [grid_and_list]({{examples}}/layout/grid_and_list)
-</div>
-<div class="col-lg-6" markdown="1">
-  {% asset ui/layout/gridview-count-flutter-gallery.png class="mw-100"
-      alt="A 2 column grid with footers" %}
-  {:.text-center}
-
-  Uses `GridView.count` to create a grid that's 2 tiles wide in portrait mode,
-  and 3 tiles wide in landscape mode. The titles are created by setting the
-  `footer` property for each [GridTile][].
-
-  **Dart code:** [grid_list_demo.dart]({{demo}}/material/grid_list_demo.dart)
-  from the [Flutter Gallery][]
-</div>
-</div>
+### Sizing widgets can be tricky
 
-<?code-excerpt "layout/grid_and_list/lib/main.dart (grid)" replace="/\GridView/[!$&!]/g;"?>
-```dart
-Widget _buildGrid() => [!GridView!].extent(
-    maxCrossAxisExtent: 150,
-    padding: const EdgeInsets.all(4),
-    mainAxisSpacing: 4,
-    crossAxisSpacing: 4,
-    children: _buildGridTileList(30));
+When a group of widgets are too large for a row, a striped pattern appears along the affected edge. Now, run the code and then check the output.
 
-// The images are saved with names pic0.jpg, pic1.jpg...pic29.jpg.
-// The List.generate() constructor allows an easy way to create
-// a list when objects have a predictable naming pattern.
-List<Container> _buildGridTileList(int count) => List.generate(
-    count, (i) => Container(child: Image.asset('images/pic$i.jpg')));
-```
+<iframe src="https://dartpad.dartlang.org/experimental/embed-new.html?id=e857a9b4e8584dfb06e9105fb3d49569&fw=true" width="100%" height="400px"></iframe>
 
-<hr>
 
-### ListView
+After running the code, you see one of three images appear in Demo1. At the image's right edge, you also see a black, rectangular box with vertical text reading "RIGHT OVERFLOWED BY 522 PIXELS." You might ask, "Where's the striped pattern?" Well, if you haven't checked the output already, once again, check it. The output tells you that the black, rectangular box is the striped pattern. Moreover, the output tells you how to fix the problem: use an Expanded widget. 
```

Comment:
Let's name the examples differently than Demo1, Demo2, etc. Let's use something a bit more descriptive.

Reply:
You got it.